inc/saitypes.h

```diff
@@ -594,11 +594,16 @@ typedef struct _sai_attribute_t
 
 typedef enum _sai_bulk_op_type_t
```

Comment:
Yes, missed correcting this name earlier. But since we are at it, better to fix it before it gets much widely utilized.